code/BuddyTrackerBasic/src/main.cpp

```diff
@@ -99,25 +102,28 @@ void sendPacket(BT_Packet packet){
 // currently adds unknown buddies
 // shows everyone
 void updateBuddy(uint64_t UUID, uint16_t lat_partial, uint16_t lng_partial){
-    uint8_t index = findBuddyIndex(UUID);
+    uint8_t index = findBuddyBy(UUID);
```

Comment:
findBuddy returns an index, it does not find by index. Can change name to just findBuddyBy since UUID, the key, is the first param name. Or change to findBuddyByUUID.